extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/dart/controller/DartTableInputSpecSlicer.java

```diff
@@ -218,6 +252,26 @@ static Set<DartQueryableSegment> findQueryableDataSegments(
     );
   }
 
+  private DartQueryableSegment toDartQueryableSegment(
+      ServerSelector serverSelector,
+      Interval interval,
+      ToIntFunction<ServerSelector> toWorkersFunction
+  )
+  {
+    final DataSegment dataSegment = serverSelector.getSegment();
+    if (serverSelector.isRealtimeSegment()) {
+      final Set<DruidServerMetadata> servers = serverSelector.getAllServers(cloneQueryMode)
```

Comment:
style nit: this stack of calls starts too far to the right and so it leads to very long lines with lots of dead air on the left-hand side. It would look nicer to start it on the next line and format it like this:

```
      final Set<DruidServerMetadata> servers =
          serverSelector.getAllServers(cloneQueryMode)
                        .stream()
                        .filter(druidServerMetadata -> segmentSource.getUsedServerTypes()
                                                                    .contains(druidServerMetadata.getType()))
                        .collect(Collectors.toSet());
```